02.calendar/cal.rb

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env ruby
+require 'date'
+require 'optparse'
+
+params = ARGV.getopts("m:", "y:")
+
+month = params["m"] ? params["m"].to_i : Date.today.month
+year = params["y"] ? params["y"].to_i : Date.today.year
+
+start_day = Date.new(year, month, 1)
+end_day = Date.new(year, month, -1)
+
+day_of_week_num = start_day.wday
+
+# 日数 + 必要な要素数の配列を作る
+head_space = ["　"] * day_of_week_num
+days = (start_day..end_day).map { |date| date.strftime("%e") }
+month_days_array = head_space + days
```

Comment:
```suggestion
month_days = head_space + days
```

複数形にすればそれだけで配列であることが伝わるので`_array`は不要です